pkg/instance/ttlDestroyConsumer_test.go

```diff
@@ -40,12 +45,38 @@ func (s *ttlSuite) TestConsumeDeletesInstance() {
 	}, s.timeout, time.Second)
 }
 
+type userClient struct {
+	mock.Mock
+}
+
+func (u *userClient) FindUserById(token string, id uint) (*models.User, error) {
+	return nil, nil
+}
+
+func (u *userClient) FindGroupByName(token string, name string) (*models.Group, error) {
+	return nil, nil
+}
+
+func (u *userClient) SignIn(username, password string) (*models.Tokens, error) {
+	return &models.Tokens{
```

Comment:
this will then need to change to something like the `Delete` implementation below